src/bridge/IrcBridge.ts

```diff
@@ -86,7 +86,7 @@ export class IrcBridge {
     private memberListSyncers: {[domain: string]: MemberListSyncer} = {};
     private joinedRoomList: string[] = [];
     private dataStore!: DataStore;
-    private startedUp = false;
+    private bridgeState: "not-started"|"running"|"killed" = "not-started";
```

Comment:
I know it's functionally identical, but any reason why it's not a proper typescript enum?

Reply:
I tend to only use enums when it's getting used more widespread than this, I suppose it's a question of style. TS having strict string checks is good nuff for me.